lenses/tests/test_krb5.aug

```diff
@@ -1071,3 +1071,37 @@ test Krb5.lns get dbmodules_test =
     }
     { "db_module_dir" = "/some/path" }
   }
+
+(* [plugins] test *)
+let plugins_test = "[plugins]
+    clpreauth = {
+        module = mypreauth:/path/to/mypreauth.so
+    }
+    ccselect = {
+        disable = k5identity
```

Comment:
`disable` and `enable_only` might need to be a `seq` as it can be specified multiple times https://web.mit.edu/kerberos/krb5-1.18/doc/admin/host_config.html